tdrs-backend/tdpservice/parsers/schema_defs/tanf/t2.py

```diff
@@ -113,7 +113,7 @@
                     condition_function=validators.oneOf((1, 2)),
                     result_field="WORK_PART_STATUS",
                     result_function=validators.oneOf(
-                        ["01", "02", "05", "07", "09", "15", "17", "18", "19", "99"]
+                        ["01", "02", "05", "07", "09", "15", "17", "18", "19", "  "]
```

Comment:
```suggestion
                        ["01", "02", "05", "07", "08","09", "15", "17", "18", "19", "99"]
```

Reply:
this was my mistake. also added 08 per [this comment](https://github.com/raft-tech/TANF-app/issues/2681#issuecomment-1979278047)